tests/test_base.cpp

```diff
@@ -865,7 +873,8 @@ nodes_data::ptr start_nodes(start_nodes_config &start_config) {
 		for (size_t j = 0; j < start_config.configs.size(); ++j) {
 			if (j == i)
 				continue;
-
+			if (!config.custom_remote_indexes.empty() && !config.custom_remote_indexes.count(j))
```

Comment:
If `config.custom_remote_indexes` is empty `.count(j)` will took same time.
```suggestion
			if (!config.custom_remote_indexes.count(j))
```

Reply:
`config.custom_remote_indexes.empty()` is a special case when we invert the logic: empty `custom_remote_indexes` means old default behavior: to connect to all servers. If indexes aren't empty, we provide new logic. Suggested change won't work.

I developed it that way because I want to simplify configuring for all old tests: we just skip filling of the parameter, and get old (default) behavior.

I can use `boost::optional<std::unordered_set<size_t>> custom_remote_indexes`, it'll make the code more understandable. How do you think?

Reply:
I reworked to `boost::optional`.